superset/viz.py

```diff
@@ -102,6 +102,11 @@
     "size",
 ]
 
+# This regex is to get user defined filter column name, which is the first param in the filter_values function.
+# see the definition of filter_values template:
+# https://github.com/apache/superset/blob/24ad6063d736c1f38ad6f962e586b9b1a21946af/superset/jinja_context.py#L63
+FILTER_VALUES_REGEX = re.compile(r"filter_values\(['\"](\w+)['\"]\,")
```

Comment:
Maybe add the Jinja brackets as well?

```suggestion
FILTER_VALUES_REGEX = re.compile(r"""\{\{\s*filter_values\(['"](\w+)['"]\)\s*\}\}""")
```

Reply:
no. define filter_values function could use another syntax without brackets:

```
...
{%- set f_metric = f_metric or filter_values("sel_metric", "no_sel_metric_specified") -%}
{%- set f_aggregation = f_aggregation or filter_values("sel_aggregation", "no_aggregation_specified") -%}
SELECT
version AS sel_fcst_version
FROM
core_guest.active_booker_metrics_forecast_countries
WHERE
metric = '{{ f2t(f_metric) }}'
...
```